### ISSUE TITLE
Event

### ISSUE THREAD

**Issue by topninja** [nmpribeiro/vuera-ts#9]
I want to call react's event from vue component
I tried like this https://github.com/akxcv/vuera/issues/54 but not work.

- In React
<any-vue-component on={{'loading': e => console.log(e)}}></any-vue-component>

- In vue component
mounted() {
     this.$emit('loading', 123)
}
so when page loads, console should print 123 but there's no log.
How to emit event with object or variable from vue to react?

**Comment by tachyon-ops** [comment]
Yes, I think the VUE component doesn't understand how to transfer the `loading` to the React component. But your code is a bit confusing to me, would it be possible for you to setup a simple project showing the issue? I can try to work it out and see what you want to do with `this.$emit('loading', 123)`
